tests/units/classes/mock/streams/fs/directory.php

```diff
@@ -25,7 +25,7 @@ public function testMkdir()
 			->then
 				->boolean(mkdir($directory, 0777))->isTrue()
 				->integer($directory->getPermissions())->isEqualTo(0777)
-				->boolean(mkdir($directory, 0777))->isFalse()
+				->boolean(@mkdir($directory, 0777))->isFalse()
```

Comment:
Because a failed `mkdir` would likely emit an error. We don't care about the error here, we just want to assert on the return value.